On Fuji-U7xxx the jack-plug events are received very early (jack half plugged).
When the plug is inserted too slowly, the headset-microphone recording might
start before the plug is in place, leading to permanent random noise. This patch
does not fix but reduces the problem by waiting 1s before starting the
recording.

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index d481989..febadf0 100644
--- src/linux/sound/pci/hda/patch_realtek.c
+++ src/linux/sound/pci/hda/patch_realtek.c
@@ -5774,6 +5774,7 @@ static void alc_update_headset_mode_hook(struct hda_codec *codec,
 static void alc_update_headset_jack_cb(struct hda_codec *codec,
 				       struct hda_jack_callback *jack)
 {
+	msleep(1000);
 	snd_hda_gen_hp_automute(codec, jack);
 	alc_update_headset_mode(codec);
 }
